### DISCUSSION TITLE
Proper way to disable dark mode in TW3

### DISCUSSION THREAD

**Discussion by the-kro** [tailwindlabs/tailwindcss#6351] [Help] [answered]
Is setting dark to 'class' the correct way to disable dark mode in TW3?  I am using DaisyUI, which has it's own dark mode implementation.  In TW2 I just set dark to false.

**Comment by NovusOpus** [comment]
[Check the Dark Mode Section in the Upgrade Guide](https://tailwindcss.com/docs/upgrade-guide#remove-dark-mode-configuration)

**Comment by lukewarlow** [comment, marked as answer]
Dark mode being on by default doesn't affect your CSS. It only does something when you have `dark:example-class` in your code. 

Unless DaisyUI relies on `dark:example-class` syntax?

**Reply by NovusOpus** [reply]
They use something like this: `<div data-theme="dark" ></div>`, [https://daisyui.com/docs/default-themes](https://daisyui.com/docs/default-themes)

**Reply by the-kro** [reply]
> Dark mode being on by default doesn't affect your CSS. It only does something when you have `dark:example-class` in your code.
> 
> Unless DaisyUI relies on `dark:example-class` syntax?

I guess that makes sense with it being JIT and all. Thanks for the clarification.

**Reply by jessieconnralphsam** [reply]
> They use something like this: `<div data-theme="dark" ></div>`, https://daisyui.com/docs/default-themes

thanks

**Comment by janat08** [comment]
My browser seems to have applied dark mode from daisyui automatically (which also happened on ios), until I specified light theme manually.

**Reply by devisreal** [reply]
Please how did you set this

**Comment by RomanKrasavtsev** [comment]
If you only want the default light theme, set themes config to false.

```
module.exports = {
  //...
  daisyui: {
    themes: false,
  },
}
```
https://daisyui.com/docs/themes/#-2

**Comment by fini** [comment]
The following will disable the dark theme, however it's not very well documented.

```
daisyui: {
  darkTheme: false,
},
```

**Reply by MUSHANIF** [reply]
thanks!

**Comment by naranyala** [comment]
or simply include only light themes

```js
// tailwind.config.js
export default {
	daisyui: {
		themes: ["light"]
	}
}
```

**Comment by archbong** [comment]
#If you only want the default light and dark themes, set themes config to false.
`module.exports = {
  //...
  daisyui: {
    themes: false,
  },
}`
If you don't want to include any themes and disable all colors, set themes config to an empty array.
`module.exports = {
  //...
  daisyui: {
    themes: [],
  },
}
`
If you're building with nextjs, put it in your tailwind.config.ts
